### ISSUE TITLE
Post: no que trabalham

### ISSUE THREAD

**Issue by nazareno** [analytics-ufcg/vereadorescg-site#4]
O trabalho típico de um vereador

**Comment by augustoqm** [comment]
Aqui poderia caber algo das comissões tecnicas, pq elas definem bem o foco tematico dos candidatos.

**Comment by nazareno** [comment]
Você consegue estruturar quantos meses cada vereador esteve na câmara, @augustoqm ? Tentei dividir a atuação dos vereadores pela quantidade de meses em que eles apresentaram ementas, mas na verdade alguém que concentre a atuação em poucos meses e nunca mais atue (embora esteja lá) fica com um bom número. Então precisava saber em que meses eles estavam lá pra poder fazer algo como ementas / mês.

**Comment by augustoqm** [comment]
Sim. Vou adicionar a informação dos afastamentos e posses no BD e podemos adicioná-la na API.

**Comment by augustoqm** [comment]
@nazareno dei uma geral nessa página para ela ficar dentros dos padrões de publicação. Já que você vai focar só nas quantidades e tipos (leg ou adm) outro argumento possível, além dos meses trabalhados, é explicar que os candidatos que fizeram parte da mesa diretora tem naturalmente mais participações em atos legislativos.

**Comment by augustoqm** [comment]
https://nazareno.github.io/vereadorescg-site//2016/09/no-que-trabalham.html

**Comment by nazareno** [comment]
"candidatos que fizeram parte da mesa diretora tem naturalmente mais participações em atos legislativos" --> Como é isso, @augustoqm ?

**Comment by augustoqm** [comment]
Nesse momento é um feeling nazareno, mas acredito que ele veio de alguma análise... Acredito que vi Pimentel Filho com muitos atos legislativos e ele está na mesa e tal... Mas pode ser porque ele é antigo na casa.
